contrib/win32/openssh/copyssl.ps1

```diff
@@ -0,0 +1,44 @@
+﻿param (
+        [string]$sourceUrl = "https://github.com/PowerShell/libressl/releases/latest/",
+        [string]$zipDir,
+        [string]$destPath)
+
+If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
+#workaround that $PSScriptRoot is not support on ps version 2
+if([string]::IsNullOrEmpty($zipDir))
+{
+    $zipDir = $PSScriptRoot
+}
+if([string]::IsNullOrEmpty($destPath))
+{
+    $destPath = $PSScriptRoot
+}
+if (Test-Path (Join-Path $destPath "LibreSSL"))
+{
+    return
+}
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 -bor `
+                                                  [Net.SecurityProtocolType]::Tls11 -bor `
+                                                  [Net.SecurityProtocolType]::Tls
+    
+    $request = [System.Net.WebRequest]::Create($sourceUrl)
+    $request.AllowAutoRedirect = $false
+    $request.Timeout = 30000; #30 sec
```

Comment:
Increase this to a minute to be on safe side.

Reply:
done